### ISSUE TITLE
entity graphic coordinates in a separate table

### ISSUE THREAD

**Issue by nnhjy** [spine-tools/Spine-Toolbox#2975]
The graphic position of an entity is now stored as the entity parameters `x` and `y`, but this information is not directly related to the model.

To keep the `parameter value` table a bit tidier, we could store the `x` and `y`s in a separate table, just like the `active` value in `Entity alternative`.

**Comment by jkiviluo** [comment]
Yes, there's a plan to move GIS data to a separate table in the DB. Where to put the data in the DB editor is still bit unclear, but it could be another tab.
